public/app/features/explore/NoDataSourceCallToAction.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import { css } from 'emotion';
+import { ThemeContext, ExtraLargeLinkButton, CallToActionCard } from '@grafana/ui';
+
+export const NoDataSourceCallToAction = () => {
+  const theme = useContext(ThemeContext);
+
+  const message = 'Please add a datasource that supports Explore (e.g., Prometheus)';
+  const footer = (
+    <>
+      <i className="fa fa-rocket" />
+      <> ProTip: You can also define data sources through configuration files. </>
+      <a
+        href="http://docs.grafana.org/administration/provisioning/#datasources?utm_source=grafana_ds_list"
```

Comment:
Nice protip! Also +1 on the utm markers.

Reply:
This is actually copied from data sources list. we need to update the utm_source here.